src/main/java/mate/controller/CreateDriverController.java

```diff
@@ -0,0 +1,29 @@
+package mate.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import mate.lib.Injector;
+import mate.model.Driver;
+import mate.service.DriverService;
+
+public class CreateDriverController extends HttpServlet {
+    private final Injector injector = Injector.getInstance("mate");
```

Comment:
```suggestion
    private static final Injector injector = Injector.getInstance("mate");
```
injector may be static